debugger/src/main.rs

```diff
@@ -1,3 +1,6 @@
+// Bad things happen to the VSCode debug extenison and debug_adapter if we panic at the wrong time.
+#![deny(clippy::unwrap_used, clippy::panic, clippy::expect_used)]
+#![allow(clippy::or_fun_call)]
```

Comment:
@Yatekii I have changed to a warn, but cannot see how to avoid the allow here (maybe your comment got lost). Can you provide a hint please?

Reply:
Use `unwrap_or_else ` instead of `unwrap_or` and `or_else` instead of `or`, etc. If you point me to the issue you are having, I can tell it more specifically :)

Reply:
Ah ... thank you. I removed the offending `allow` from both places you flagged it.